### PR TITLE
Desktop version Using media queries

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
-Using media queries, I configured the desktop version of the portfolio.
- I used grid and flexbox to place elements correctly.
-

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -483,10 +484,225 @@ textarea {
...
+}
+
+.aboutContact {
+  display: none;
+}
+
+@media only screen and (min-width: 992px) {
+  .container3 {
+    width: 100%;
+    height: 800px;
```
Comment (Wes-Isaac) [inline style.css:510]: - [ ] Great job implementing media query, however, there is a small gap before the body which is not according to the style, please fix that.
- [ ] In the desktop version there shouldn't be a hamburger menu, Please make sure that it's not displayed in the desktop version.
- [ ] Please follow the spacing provided in the figma design in the navbar at the end there are overlapping elements and also the postal icon is not there please fix it.

### Your implementation

image

## Figma's design
image

File: style.css [hunk 2]
```diff
@@ -483,10 +484,225 @@ textarea {
...
+    left: 60%;
+    top: 140%;
+  }
+
+  .language {
+    position: absolute;
+    left: 60%;
+    top: 180%;
+    padding-left: 20px;
+  }
```
Comment (Wes-Isaac) [inline style.css:622]: - [ ] This block is moving up and down when the vertical length is varied. Kindly give it a proper position specified in the design.

### Your implementation

image

File: index.html [hunk 3]
```diff
@@ -1,116 +1,88 @@
...
-          sign-ups required. has been the industry's standard
-        </p>
+    <div class="illus">
+      <img src="picsNew/backgrnd.png" alt="illustration" />
+    </div>
+  </section>
+  <section>
+    <div class="works">
+      <h1 class="recent">MY RECENT WORKS</h1>
+      <div class=""><img src="pics/image1.png" alt=""></div>
```
Comment (Wes-Isaac) [inline index.html:63]: - [ ] This part is missing some things from the design for the desktop version. Kindly fix that.
- [ ] Please make sure you're using the same font.

 ### Your implementation 

image

### Figma's Design

image

File: style.css [hunk 4]
```diff
@@ -483,10 +484,225 @@ textarea {
...
+
+  .description {
+    font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
```
Comment (Wes-Isaac) [inline style.css:654]: - [ ] Great job there are a few things missing, the image is moving up and down please fix the position of that image and Kindly use the same font, spacing for all elements specified in the design.

### Your implementation

image

### Figma's Design
image

File: style.css [hunk 5]
```diff
@@ -483,10 +484,225 @@ textarea {
...
+
+  .skills {
+    padding-top: 7%;
+    list-style: none;
+  }
+
+  .frameworks {
+    padding-top: 8%;
+    list-style: none;
+  }
```
Comment (Wes-Isaac) [inline style.css:670]: - [ ] Please mind the space(margin & padding) on the framework section, they are not similar to the design. Kindly fix that.

### Your implementation

image

### Figma's Design
image

File: style.css [hunk 6]
```diff
@@ -148,9 +140,9 @@
   padding-bottom: 20px;
 }
 
-.illus {
+.illust {
   position: absolute;
-  right: 70%;
+  right: 60%;
   top: 30%;
 }
```
Comment (Wes-Isaac) [inline style.css:147]: - [ ] While you have made a great effort positioning this element, the image is broken for the mobile version.

### Your implementation

image

### Figma's Design
image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @MICHAELMUNAVU83, @BasitKorede, 

Impressive work so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!

### 📝👌 Your project highlights 🔥😎

- Linters are on point ✔
- Splendid work with the README file 👀
- Proper use of GitHub flow 👌

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 7]
```diff
@@ -483,10 +484,225 @@ textarea {
...
+  }
+
+  .lastImage {
+    margin-left: 9%;
+  }
+
+  .projectDiv {
+    display: flex;
+    flex-wrap: wrap;
+  }
```
[the remark this answers is not in the document]
Comment (nicupop729) [reply style.css:838]: **Here is what the figma template looks like:**
image

**And here is what your project section looks like:**  
image

- [ ] Kindly address this design with better use of Flexbox or Grid.

- [ ]  And please arrange the technologies to be situated in the middle of the boxes, and not top left as they are in now

File: index.html [hunk 8]
```diff
@@ -177,145 +190,122 @@ <h1 class="recent">MY RECENT WORKS</h1>
...
+
+    <p class="lastImage">
+      <img src="picsNew/Shape.png" alt="last shape">
+    </p>
+
+
+
+
+
+
```
Comment (nicupop729) [inline index.html:306]: - [ ] Please remove the empty lines from 301 to 306.

File: style.css [hunk 9]
```diff
@@ -483,10 +567,356 @@ textarea {
...
+    line-height: 52px;
+    display: grid;
+    grid-template-columns: 440px;
+    padding-bottom: 60px;
+    color: #262626;
+  }
+
+  .works {
+    width: 100%;
+  }
```
Comment (AnselemOdims) [inline style.css:707]: - [ ] As already pointed out by the previous reviewer, please make sure your design adheres to the original design guidelines(fonts, colors, margins, padding etc). Kindly compare your design to the original design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
- [ ] As can be seen, yours is inverted. The list of technologies and the buttons do not align with the original design. Please implement these changes so your design can look identical to the design

### Your Design
image

### Original Design
image

File: style.css [hunk 10]
```diff
@@ -202,63 +205,115 @@
...
 }
 
-.backProjects {
+.backProjects1 {
+  height: 386px;
+  width: 327px;
+  left: 10px;
+  background-position: center;
+  background-image: url("pics/back.png");
+}
```
Comment (AnselemOdims) [inline style.css:282]: - [ ] As already pointed out by the previous reviewer, please make sure the list of technologies (HTML, CSS, Ruby, JavaScript) on each project card aligns properly and sits in the middle of its box.

### Your Design
image

### Original Design
image

File: style.css [hunk 11]
```diff
@@ -483,10 +567,356 @@ textarea {
...
+    position: absolute;
+    left: 1%;
+    top: 205%;
+    padding-top: 20px;
+  }
+
+  .seeInfo {
+    left: 1%;
+    top: 215%;
+    margin-left: 30px;
```
Comment (AnselemOdims) [inline style.css:815]: - [ ] As stated earlier by the previous reviewers, kindly make sure the contact form matches the original design. At the moment, this is what it looks like 

### Your  Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @MICHAELMUNAVU83, 
Good job implementing some of the changes suggested by previous reviewers.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
- [ ] As already suggested by previous reviewers, I would also like to suggest avoiding using `position` to arrange elements(except in special occasions). It would be better to use `grid` and/or `flex` when positioning elements.

_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!. Kindly tag me using @AnselemOdims for a quicker response or reach out to me on slack.

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**